modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v1/form_data.rb

```diff
@@ -59,11 +59,11 @@ def soc_opt_in
         end
 
         def signature
-          first_last = [first_name('Veteran'), last_name('Veteran')]
-          formatted_name = first_last.map(&:presence).compact.map(&:strip).join(' ')
-          auth_statement = 'signed by digital authentication to api.va.gov'
+          first_last = [first_name('Veteran')&.[](0...90), last_name('Veteran')&.[](0...90)]
```

Comment:
Do you think it would be useful to have a couple specs around this truncation logic?